From ec2e9eac245df6d69dda30c13fef06be4caaef46 Mon Sep 17 00:00:00 2001
From: is-qian <tangruiqian17@gmail.com>
Date: Sat, 27 Apr 2024 21:27:57 +0800
Subject: [PATCH] compatible for pi-gen

---
 scripts/reTerminal.sh | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/scripts/reTerminal.sh b/scripts/reTerminal.sh
index dc09545..bfafc5c 100755
--- a/scripts/reTerminal.sh
+++ b/scripts/reTerminal.sh
@@ -18,14 +18,10 @@ MOD_PATH=`pwd`/modules
 RES_PATH=`pwd`/extras/reTerminal/resources
 
 # RPI
-CFG_PATH=/boot/config.txt
-CLI_PATH=/boot/cmdline.txt
-OVERLAY_DIR=/boot/overlays
+CFG_PATH=/boot/firmware/config.txt
+CLI_PATH=/boot/firmware/cmdline.txt
+OVERLAY_DIR=/boot/firmware/overlays
 BLACKLIST_PATH=/etc/modprobe.d/raspi-blacklist.conf
-# Ubuntu or Raspbian bookworm and later
-[ -f /boot/firmware/config.txt ] && CFG_PATH=/boot/firmware/config.txt
-[ -f /boot/firmware/cmdline.txt ] && CLI_PATH=/boot/firmware/cmdline.txt
-[ -d /boot/firmware/overlays ] && OVERLAY_DIR=/boot/firmware/overlays
 
 # DISTRO INFO
 DISTRO_ID=$(lsb_release -is)
-- 
2.40.1

